### DISCUSSION TITLE
My two cents on the "Open Sourcerer" and "Heart on Your Sleeve" achievements requirements

### DISCUSSION THREAD

**Discussion by rickstaa** [community/community#55384] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I just noticed that a user posted the requirements for getting the "Heart On Your Sleeve" and "Open Sourcerer" achievements on the non-official Achievements page (see https://github.com/Schweinepriester/github-profile-achievements/pull/738). I double-checked with the GitHub support and the requirements for achieving currently planned as:

`Open Sourcerer`: User had PRs merged in multiple public repositories.
`Heart on your Sleeve`: React to something on GitHub with a ❤️ emoji.

Since the support asked me if I would provide feedback on these requirements in the https://github.com/orgs/community/discussions and start a discussion to help GitHub's @ProductTeam to get feedback on these requirements, here are my two cents. 

In my opinion, the requirements of the `Open Sourcerer` achievements are great since it incentivises people to contribute to OS projects :rocket:! It would be great to also have different levels to show how much PRs a person did for OS projects. I, however, can see how this will be too similar to the Pull Shark achievements.

For the `Heart on your Sleeve` achievement requirements, although I love to see more hearts on GitHub, I think the requirements of this achievement are too easy. I would love to see extra requirements added. For example:
1. React to something with a :heart: **AND**
2. Donate to an OS project **AND**
3. Answered a discussion post in an OS project.

Further, since the naming of the achievement is similar to the pride moth slogan, you could also link the release of this badge to that month (see https://github.blog/2017-05-09-get-ready-for-pride-month-with-our-new-github-pride-and-trans-shirts/). For example, only award the achievement for hearts put on GitHub during this month. Hope my feedback helps!

**Comment by abinth11** [comment]
They mentioned that by just reacting a heart we will get that. But i didn't get it

**Reply by rickstaa** [reply]
I had the same problem. Sounds a bit too easy to me 😅.

**Reply by SkyrocketStan** [reply]
I didn't get it too.

**Reply by WenhaoYou1** [reply]
❤️

**Reply by mzums** [reply]
❤️

**Reply by a-rustacean** [reply]
❤️

**Reply by muze-pus** [reply]
❤️

**Reply by green52199** [reply]
❤️

**Reply by kartiksoni01** [reply]
❤️

**Reply by atilagulers** [reply]
❤️

**Reply by a-rustacean** [reply]
:heart:

**Reply by EMESTAY** [reply]
❤️

**Reply by Sidmaz666** [reply]
❤️

**Reply by ReznorsRevenge1** [reply]
❤️

**Reply by schikimikiviki** [reply]
❤️

**Reply by T1rth1** [reply]
❤️

**Reply by Soumya-Chakraborty** [reply]
♥️

**Reply by rayc2045** [reply]
❤️

**Reply by sahilpanghal** [reply]
❤️

**Reply by Kir4R00t** [reply]
❤️

**Reply by Parq254** [reply]
❤️

**Reply by shirabiton** [reply]
❤️

**Reply by BrianCodeDev** [reply]
nice

**Reply by the-zoomeee** [reply]
❤️

**Reply by GonzaloMB** [reply]
❤️

**Reply by ryzmdn** [reply]
❤️

**Reply by Seba3995** [reply]
❤️

**Reply by yihong1120** [reply]
❤️

**Reply by imran-baitham** [reply]
❤️

**Reply by Poo-wei-chien** [reply]
❤️

**Reply by Asm3515** [reply]
> They mentioned that by just reacting a heart we will get that. But i didn't get it

❤️

**Reply by smarenkov** [reply]
❤

**Reply by mm-kishanp** [reply]
❤️

**Reply by yihsunphys** [reply]
❤

**Reply by Bitus116** [reply]
❤

**Reply by KavinMadusanka** [reply]
❤️

**Reply by kiaSso** [reply]
❤️

**Reply by Kishara98** [reply]
❤️

**Reply by AE-Hertz** [reply]
️❤️

**Reply by thenocturnaldevgypsy** [reply]
️❤️

**Reply by debanjandhara** [reply]
❤

**Reply by Utkarshgola** [reply]
❤️

**Reply by KuperDude** [reply]
❤️

**Reply by RainerNsa** [reply]
❤️

**Reply by Mateen26** [reply]
❤️

**Reply by srawan071** [reply]
❤️

**Reply by robinrgc** [reply]
❤️

**Reply by pbiology** [reply]
❤️

**Reply by AronSoto** [reply]
❤️

**Reply by jjrh92** [reply]
❤️

**Reply by Langl3y** [reply]
❤️

**Reply by nsvetoslav** [reply]
❤️

**Reply by unreturned** [reply]
❤️

**Reply by Neha-Singh-j** [reply]
> They mentioned that by just reacting a heart we will get that. But i didn't get it

❤️

**Reply by 5pence** [reply]
❤️

**Reply by AndreMatheusFil** [reply]
❤️

**Reply by onthecrow** [reply]
❤️

**Reply by nuekkis** [reply]
❤️

**Reply by StoynovAngel** [reply]
> ❤️
❤️

**Reply by damianczer** [reply]
❤️

**Reply by hitesh-mori** [reply]
❤️

**Reply by YonisDev** [reply]
❤️

**Reply by KoaDev** [reply]
❤️

**Reply by EmielDavid** [reply]
❤️

**Reply by jsporna** [reply]
❤️

**Reply by Niaro** [reply]
❤️

**Reply by ZouhairSA** [reply]
❤️

**Reply by ThEMueA** [reply]
❤️:3

**Reply by barthollomew** [reply]
❤️

**Reply by 4x2vk** [reply]
❤️

**Reply by xarlizard** [reply]
❤️

**Reply by pixsphet** [reply]
❤️

**Reply by GreyGhost27-art** [reply]
❤️

**Reply by RafaDEV99** [reply]
❤️

**Reply by H-Hp** [reply]
❤️

**Reply by ShahzaibAhmad05** [reply]
❤️

**Reply by yang-liang** [reply]
❤️

**Reply by Finnimonius** [reply]
❤️

**Reply by TheChroniclerr** [reply]
❤️

**Reply by rafaelmckes-boop** [reply]
❤️

**Reply by Hugo-Jurriaans** [reply]
❤️

**Reply by JordanVera** [reply]
❤️

**Reply by BluntSam** [reply]
❤️

**Reply by andriiafanasiev** [reply]
❤️

**Reply by sunaynatalreja** [reply]
❤️

**Reply by ComfyPenguin** [reply]
❤️

**Reply by mihir-28** [reply]
❤️

**Reply by MattyTD** [reply]
❤️

**Reply by bari199** [reply]
❤

**Reply by jaydendancer12** [reply]
❤️

**Reply by NicolasPauferro** [reply]
❤️

**Reply by kenjivafe** [reply]
❤️

**Reply by nishita-dev199** [reply]
❤️

**Reply by MecGlandorff** [reply]
❤️

**Reply by devnazir** [reply]
❤️

**Reply by Joehawkk** [reply]
❤️

**Reply by sohje** [reply]
❤️

**Reply by LuisAhumadaMartens** [reply]
❤️

**Reply by Musxeto** [reply]
❤️

**Reply by biozal** [reply]
<3

**Reply by KSAGlory** [reply]
❤️

**Reply by marco-crest** [reply]
❤️

**Reply by aarmam** [reply]
❤️

**Reply by kerberjg** [reply]
❤️

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀

**Reply by saucecontrol** [reply]
Clearly this has not had the desired effect, as the low quality, nonsense, and spam posts continue.
![image](https://github.com/community/community/assets/7929102/d2e7b6b6-9311-42bd-bed6-2636b9bbd2e8)

Even if this change were to have been effective, it would simply shift the nonsense to other orgs/projects in GitHub.  The problem is the 'Achievements' themselves.  When you make childish rewards, you invite childish behavior.  This has no place in GitHub.

**Reply by WallysonMachadoDeLima** [reply]
❤️

**Reply by xmaxrayx** [reply]
can we remove achievements? this isn't social media

**Reply by yihong1120** [reply]
❤️

**Reply by imran-baitham** [reply]
❤️

**Reply by Poo-wei-chien** [reply]
❤️

**Reply by smarenkov** [reply]
❤

**Reply by debanjandhara** [reply]
❤

**Reply by joaomota59** [reply]
❤️

**Reply by oscar-mx** [reply]
❤️

**Reply by VichuA2** [reply]
❤️

**Reply by unrays** [reply]
❤️

**Reply by andriiafanasiev** [reply]
❤️

**Reply by Jah-yee** [reply]
❤️

**Reply by murtiunlimited** [reply]
❤️

**Reply by LuisAhumadaMartens** [reply]
❤️

**Reply by Musxeto** [reply]
❤️

**Comment by AbuBakkarRaihan** [comment]
Great requirements for "Open Sourcerer" and "Heart on Your Sleeve"! I especially like the focus on encouraging open-source contributions and community engagement. Keep up the good work!

**Comment by xmaxrayx** [comment]
how I can unsub this from my email? this just annoying.

**Reply by xmaxrayx** [reply]
wow really you need to log-in to unseb? thx github for more unconvince.

**Reply by ghost** [reply]
❤

**Reply by jaydendancer12** [reply]
❤️

**Comment by anumeetkumar** [comment]
Hello there

**Reply by MindfulLearner** [reply]
Hello!

**Reply by cdntunderstand** [reply]
Hello I am here ready to learn

**Comment by MindfulLearner** [comment]
I love open source!

**Comment by http-shubjit** [comment]
I love open source! ❤

**Comment by huuthang201** [comment]
I love open source! ❤

**Comment by damianczer** [comment]
I love open source! ❤

**Comment by md-rifatkhan** [comment]
I love open sources

**Comment by Rony-2004** [comment]
I LOVE OPEN SOURCE ❤

**Comment by H-Hp** [comment]
I love open source! ❤

**Comment by TheChroniclerr** [comment]
I love open source! ❤

**Comment by Jah-yee** [comment]
I love open source! ❤

**Comment by imran-baitham** [comment]
𝑰 𝑳𝑶𝑽𝑬 𝑶𝑷𝑬𝑵 𝑺𝑶𝑼𝑹𝑪𝑬 ❤

**Comment by RiturajKulshresth** [comment]
I love open source! ❤

**Comment by devnazir** [comment]
I love open source! ❤

**Comment by cheesecakefatorymenu** [comment]
[Sweetgreen](https://sweetgreensrestaurant.com/) is a popular healthy food restaurant chain that focuses on serving fresh salads, grain bowls, and protein-packed meals made with natural and locally sourced ingredients. The brand is widely known for promoting healthy eating through customizable menu options that include fresh greens, roasted vegetables, chicken, avocado, quinoa, and flavorful homemade dressings.

**Comment by Mahmoud-Eltabakh** [comment]
Hello World!

**Comment by Musxeto** [comment]
I love open source! ❤
